crates/miden-objects/src/block/proposed_block.rs

```diff
@@ -439,43 +460,53 @@ fn check_batch_reference_blocks(
 ///
 /// After the batch note tree was made consistent, we insert it as a subtree into the larger block
 /// note tree.
-fn compute_block_note_tree(
+///
+/// Returns the block note tree as well as the [`OutputNote`] that each batch creates.
+fn compute_block_output_notes(
```

Comment:
Doc comments for this function need to be updated (i.e., we are no longer building a `BlockNoteTree` here.